src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1673,6 +1687,8 @@ static bool addTriggeredSubscriptions_noCache
       bool              onlyChanged        = sub.hasField(CSUB_ONLYCHANGED)? getBoolFieldF(sub, CSUB_ONLYCHANGED) : false;
       bool              blacklist          = sub.hasField(CSUB_BLACKLIST)? getBoolFieldF(sub, CSUB_BLACKLIST) : false;
       bool              covered            = sub.hasField(CSUB_COVERED)? getBoolFieldF(sub, CSUB_COVERED) : false;
+      bool              notifyOnMetadataChange = sub.hasField(CSUB_NOTIFYONMETADATACHANGE)? \
+      getBoolFieldF(sub, CSUB_NOTIFYONMETADATACHANGE) : false;
```

Comment:
Better in the same line, avoiding `\` break.

Reply:
Thank you for your suggestion, but if I add code in the same line, I exceed the limit of 120 characters, and the compliance test scenario fails.

Reply:
I understand you refer to [S9 rule](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/devel/contribution_guidelines.md#s9-line-length).

Note that that rule is in the "SHOULD" section, not in the "MUST" section and that there are existing lines exceeding that limite (L1685 is 140 chars long).

Thus, I'd suggest to use put all them in the same line and let's see if the compliance tests fails (I think it would pass, but let's see...)